api/envoy/config/filter/http/rate_limit/v2/rate_limit.proto

```diff
@@ -51,5 +51,7 @@ message RateLimit {
   // Configuration for an external rate limit service provider. If not
   // specified, any calls to the rate limit service will immediately return
   // success.
+  // TODO(ramaraochavali): Make this required as part of cleanup of deprecated ratelimit service
```

Comment:
Can you make this a doc comment so it doesn't show up rendered? Same elsewhere. See https://github.com/envoyproxy/envoy/blob/master/api/CONTRIBUTING.md

Reply:
Sure. will do. I searched for this and could n't find one.